### PR TITLE
Add MeetingReadinessCheckerConfiguration

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
Adding MeetingReadinessCheckerConfiguration to allow specify custom timeout and wait time for meeting readiness checker APIs.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes?
- Unit tests
- Manually run app checker demo and verify everything still works.
3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
Yes, modify the DefaultMeetingReadinessChecker constructor to allow an optional parameter.
4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? 
No

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-chime-sdk-js",
-  "version": "1.18.0",
+  "version": "1.18.2",
```
Comment (simmkyu) [inline package.json:3]: Patch update from `0` to `2`?

File: src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts [hunk 2]
```diff
@@ -39,7 +42,19 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   private browserBehavior: DefaultBrowserBehavior = new DefaultBrowserBehavior();
 
-  constructor(private logger: Logger, private meetingSession: MeetingSession) {}
+  constructor(
+    private logger: Logger,
+    private meetingSession: MeetingSession,
+    configuration?: MeetingReadinessCheckerConfiguration
+  ) {
+    this.configuration = new MeetingReadinessCheckerConfiguration();
```
Comment (simmkyu) [inline src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts:50]: As in `MeetingSessionConfiguration`, how about defining default values in the `MeetingReadinessCheckerConfiguration` class itself?

The constructor can be simplifed into: 
```
constructor(
    private logger: Logger,
    private meetingSession: MeetingSession,
    private configuration: MeetingReadinessCheckerConfiguration = new MeetingReadinessCheckerConfiguration()
) {}
```

Comment (ltrung) [reply src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts]: Nice suggestions. I have updated the code.

**Review by simmkyu — APPROVED** [review]
Nice!

**Review by simmkyu — DISMISSED** [review]

File: src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts [hunk 3]
```diff
@@ -39,7 +38,11 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   private browserBehavior: DefaultBrowserBehavior = new DefaultBrowserBehavior();
 
-  constructor(private logger: Logger, private meetingSession: MeetingSession) {}
+  constructor(
+    private logger: Logger,
+    private meetingSession: MeetingSession,
+    private configuration: MeetingReadinessCheckerConfiguration = new MeetingReadinessCheckerConfiguration()
```
Comment (anuranduttaroy) [inline src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts:44]: Do we want to update the documentation to have an example for using this configuration?

**Review by anuranduttaroy — APPROVED** [review]

**Pull request merged**